client/README.md

```diff
@@ -0,0 +1,73 @@
+# libSMCE command line frontend
+
+Like SMCE, this sample program requires a C++20 toolchain (for `<span>` mostly).
+
+**IMPORTANT**  
+As is, SMCE_Client is not guaranteed to work on Windows machines!
+Due to a bug that causes the running sketch to crash when sending or reciving messages on uart.
+
+
+## Build instructions
+To be able to build SMCE_client, the enviroment variable SMCE_ROOT needs to point to a current installed release of libSMCE.
+These can be found under releases on git. Or it can be build directly from the source code with the commands below.
+These should be ran in the libSMCE folder. 
+```shell
+cmake -S . -B build/
+cmake --build build/
+cmake --build build/ --config Release
+cmake --install build/ --prefix <path to installation folder>
```

Comment:
These instructions are both wrong and misplaced; this README is in a subfolder of the libSMCE project, so it stands to reason that readers have already read the top-level README